doc/cephfs/troubleshooting.rst

```diff
@@ -76,90 +76,102 @@ the progression of the read position to compute the expected time to complete.
 Avoiding recovery roadblocks
 ----------------------------
 
-When trying to urgently restore your file system during an outage, here are some
-things to do:
+Do the following when restoring your file system: 
 
-* **Deny all reconnect to clients.** This effectively blocklists all existing
-  CephFS sessions so all mounts will hang or become unavailable.
+* **Deny all reconnection to clients.** Blocklist all existing CephFS sessions,
+  causing all mounts to hang or become unavailable:
 
-  .. code:: bash
+  .. prompt:: bash #
 
      ceph config set mds mds_deny_all_reconnect true
 
   Remember to undo this after the MDS becomes active.
 
-  .. note:: This does not prevent new sessions from connecting. For that, see the ``refuse_client_session`` file system setting.
+  .. note:: This does not prevent new sessions from connecting. Use the
+     ``refuse_client_session`` file-system setting to prevent new sessions from
+     connecting to the CephFS.
 
-* **Extend the MDS heartbeat grace period**. This avoids replacing an MDS that appears
-  "stuck" doing some operation. Sometimes recovery of an MDS may involve an
-  operation that may take longer than expected (from the programmer's
-  perspective). This is more likely when recovery is already taking a longer than
-  normal amount of time to complete (indicated by your reading this document).
-  Avoid unnecessary replacement loops by extending the heartbeat graceperiod:
+* **Extend the MDS heartbeat grace period.** This avoids replacing an MDS that
+  appears "stuck" during some operation. Sometimes recovery of an MDS may
```

Comment:
s/some/an/

Reply:
This goes on the pile of final edits that will be made along with the ones I promised in https://github.com/ceph/ceph/pull/64832#discussion_r2259049315